tests/build_test.sh

```diff
@@ -238,6 +238,18 @@ function test_kernel_build_menu_cross_compilation_flags()
   assertEquals "($LINENO)" "$expected_result" "$output"
 }
 
+function test_kernel_build_clean() {
+  local expected_result
+  local output
+
+  build_config=()
+  parse_configuration "$SAMPLES_DIR/build_no_log.config" build_config
+
+  output=$(kernel_build 'TEST_MODE' --clean 2> /dev/null)
```

Comment:
Why do we need `2> /dev/null`?

Reply:
Because without the `2> /dev/null` it keeps printing this error when I run `./run_tests.sh`

![1](https://user-images.githubusercontent.com/43859386/208542925-5d78033f-00f7-4c2a-aebc-b7cf66c2562d.png)